includes/compat/subscriptions/trait-wc-payment-gateway-wcpay-subscriptions.php

```diff
@@ -706,4 +719,70 @@ public function restore_order_pay_var() {
 			$wp->query_vars['order-pay'] = $this->order_pay_var;
 		}
 	}
+
+	/**
+	 * Add card mandate options parameters to the order payment intent if needed.
+	 * Only required for the subscription creation for card issued in India.
+	 * More details https://wp.me/pc4etw-ky
+	 *
+	 * @param WC_Order $order The subscription order.
+	 * @return array Params to be included or empty array.
+	 */
+	public function maybe_add_mandate_to_order_payment( WC_Order $order ):  array {
```

Comment:
I think you may need to add some special handling for orders where the initial payment doesn't have the same value as any of the renewal payments 🤔 

I think I handled this correctly in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1981/commits/864cbb8a40dd4f76dc4c6feea40cad83f8e64f05, but I wasn't able to test it [due to API issues](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1981#issuecomment-939246499).

Kudos to @ricardo for spotting this problem in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1981#issuecomment-939208232

Reply:
Thanks @ricardo and @reykjalin for pointing it out! I missed it too.

Addressed in 2e53900